src/main/scala/mg7/dataflows/noFlash.scala

```diff
@@ -16,108 +16,117 @@ import ohnosequences.awstools.autoscaling._
-
-    DataMapping(sampleId, splitDataProcessing(params))(
-      remoteInput = Map(
-        data.mergedReads -> readsS3Resource
-      ),
-      remoteOutput = Map(
-        data.fastaChunks -> S3Resource(params.outputS3Folder(sampleId, "split"))
+  lazy val splitDataMappings =
+    splitInputs.toList.map { case (sampleId, readsS3Resource) =>
+      DataMapping(sampleId, splitDataProcessing(params))(
+        remoteInput = Map(
+          data.mergedReads -> readsS3Resource
+        ),
+        remoteOutput = Map(
+          data.fastaChunks -> S3Resource(params.outputS3Folder(sampleId, "split"))
+        )
       )
-    )
-  }
+    }
 
-  lazy val blastDataMappings = splitDataMappings.flatMap { splitDM =>
-    val sampleId = splitDM.label
+  // TODO what's this supposed to be doing?
+  lazy val blastDataMappings =
+    splitDataMappings.flatMap { splitDM =>
+      val sampleId = splitDM.label
 
-    lazy val s3 = S3.create(
-      new AWSCredentialsProviderChain(
-        new InstanceProfileCredentialsProvider(),
-        new ProfileCredentialsProvider()
+      lazy val s3 = S3.create(
+        new AWSCredentialsProviderChain(
+          new InstanceProfileCredentialsProvider(),
+          new ProfileCredentialsProvider()
+        )
       )
-    )
 
-    lazy val chunksS3Folder: AnyS3Address = splitDM.remoteOutput(data.fastaChunks).resource
-    lazy val chunks: List[S3Object] = s3.listObjects(chunksS3Folder.bucket, chunksS3Folder.key)
+      lazy val chunksS3Folder: AnyS3Address = splitDM.remoteOutput(data.fastaChunks).resource
+      lazy val chunks: List[S3Object] = s3.listObjects(chunksS3Folder.bucket, chunksS3Folder.key)
 
-    chunks.zipWithIndex.map { case (chunkS3Obj, n) =>
+      chunks.zipWithIndex.map { case (chunkS3Obj, n) =>
 
-      DataMapping(s"${sampleId}.${n}", blastDataProcessing(params))(
-        remoteInput = Map(
-          data.fastaChunk -> S3Resource(chunkS3Obj)
-        ),
-        remoteOutput = Map(
-          data.blastChunk -> S3Resource(params.outputS3Folder(sampleId, "blast") / "chunks" / s"blast.${n}.csv"),
-          data.noHitsChunk -> S3Resource(params.outputS3Folder(sampleId, "blast") / "no-hits" / s"no-hits.${n}.fa")
+        DataMapping(s"${sampleId}.${n}", blastDataProcessing(params))(
+          remoteInput = Map(
+            data.fastaChunk -> S3Resource(chunkS3Obj)
+          ),
+          remoteOutput = Map(
+            data.blastChunk -> S3Resource(params.outputS3Folder(sampleId, "blast") / "chunks" / s"blast.${n}.csv"),
+            data.noHitsChunk -> S3Resource(params.outputS3Folder(sampleId, "blast") / "no-hits" / s"no-hits.${n}.fa")
+          )
         )
-      )
+      }
     }
-  }
 
-  lazy val assignDataMappings = splitInputs.keys.toList.flatMap { case sampleId =>
+  lazy val assignDataMappings =
+    splitInputs.keys.toList.flatMap {
+      // TODO there are a lot of similar anonymous functions like this one which should use braces, so that the scope is clear
```

Comment:
There are already outer braces. I use everywhere

```scala
foo: A => B = { a =>
  ...
  getSomeB(a)
}
```

You suggest 


```scala
foo: A => B = 
  { a => {
      ...
      getSomeB(a)
    }
  }
```

Too many braces in my opinion.